apps/common/controllers/address-lookup-loop.js

```diff
@@ -109,9 +92,9 @@ module.exports = class AddressLookupLoopController extends BaseController {
   }
 
   validateField(key, req) {
-    if (req.form.values[key] === this.options.fields[`${this.field}-address-lookup`].options[0].value) {
-      return new ErrorController(`${this.field}-address-lookup`, {
-        key: `${this.field}-address-lookup`,
+    if (req.form.values[key] === this.options.fields[`${this.options.locals.field}-address-lookup`].options[0].value) {
```

Comment:
A I right in thinking this checks to see if the submitted value matches the first "null" option from the select ("please select" or similar) and throws if it matches?